sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests.WebApp/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.WebApp.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <!-- https://docs.microsoft.com/aspnet/core/test/integration-tests -->
+  
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net461' ">
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
I think this packagereference is *only* required in netcoreapp2.1. Anything higher than that gets this automatically.

Reply:
ohhhhh good catch! thanks